lib/shopify_app/engine.rb

```diff
@@ -16,6 +16,7 @@ class Engine < Rails::Engine
 
     initializer "shopify_app.assets.precompile" do |app|
       app.config.assets.precompile += %w[
+        shopify_app/app_bridge_redirect_1.30.0.js
```

Comment:
Is this needed when it's already included as part of `shopify_app/redirect.js` and the other files?